test/frozen-columns.html

```diff
@@ -51,6 +51,20 @@
           flushGrid(grid);
         });
 
+        after(() => document.documentElement.removeAttribute('dir'));
+
+        !isRTL && it('should update scrollerMetrics on dir changed to rtl after frozen columns set', done => {
+          columns[0].frozen = true;
+          grid._debouncerCacheElements.flush();
+          grid.setAttribute('dir', 'rtl');
+          listenOnce(grid.$.table, 'scroll', () => {
+            expect(grid.__scrollerMetrics).not.to.be.undefined;
```

Comment:
This checks for a very specific internal detail. Isn't there a black-box way to test the fix? Something like `expect(scrollTheGridFunction).not.to.throw(Error)`

Reply:
I have initially tried to go with `black-box` approach (both with `not.to.throw(Error)` and `spy`), but it doesn't seem to work properly. `__setNormalizedScrollLeft` doesn't throw.

I have an idea with:

```
expect(() => grid._scrollHandler()).not.to.throw(Error);
```

But it's a bit of hack as scrolling is not really happening and it's just testing handler called manually which seems to be specific internal detail.

I can try to experiment more and come up with better one.

Reply:
If it's not easily doable, not a blocker for this PR. `_scrollHandler` should get called (synchronously) if you fire a `wheel` event: https://github.com/vaadin/vaadin-grid/blob/67d6d3162a2b699a2fa6a5f2f059f075266db3c5/test/helpers.html#L111

Reply:
Thanks for the hint! Have used wheel event as an emulation of scrolling :)